app-backend/build.sbt

```diff
@@ -1,22 +1,11 @@
 import xerial.sbt.Sonatype._
-import ReleaseTransformations._
 import explicitdeps.ExplicitDepsPlugin.autoImport.moduleFilterRemoveValue
 
-addCommandAlias(
-  "gitSnapshots",
-  ";set version in ThisBuild := git.gitDescribedVersion.value.get + \"-SNAPSHOT\""
-)
-
 addCommandAlias(
   "fix",
   ";scalafix;scalafmt;scalafmtSbt"
 )
 
-git.gitTagToVersionNumber in ThisBuild := { tag: String =>
-  if (tag matches "[0-9]+\\..*") Some(tag)
-  else None
-}
-
```

Comment:
This is from the [docs](https://github.com/sbt/sbt-git#versioning-with-git) for `sbt-git`. We don't use it in any of the GeoTrellis projects, so I'm removing it from here.